openmc/deplete/results_list.py

```diff
@@ -57,15 +57,24 @@ def get_atoms(self, mat, nuc):
             Material name to evaluate
         nuc : str
             Nuclide name to evaluate
+        nuc_units : {"atoms", "atoms/b/cm", "atoms/cm^3"}, optional
+            Units for the returned concentration. Default is ``"atoms"``
+        time_units : {"s", "d"}, optional
```

Comment:
Maybe we should support `min` and `h` too as we do for the integrators?

Reply:
Sure thing!